src/helpers/themes/globalStyles.css

```diff
@@ -148,6 +148,15 @@ a {
   }
 }
 
+del {
+  text-decoration: line-through;
+  text-decoration-color: var(--oe-danger-color);
+}
+
+ins {
+  text-decoration: none;
+}
```

Comment:
maybe trial a style here? light blue (info?) underline? 

theme colour for font?

Reply:
Mock up with info color underline:

<img width="639" height="780" alt="image" src="https://github.com/user-attachments/assets/0443a742-b63f-4a2f-80e4-eea231878545" />

I'm not sure if the underline provides much benefit without the tooltip, and it does have a drawback of adding more color & noise to the verification grid.

---

<img width="646" height="772" alt="image" src="https://github.com/user-attachments/assets/547a85a3-04ec-49c8-a4f6-4048aac9d0b5" />

I think I personally like using `var(--oe-success-color)` as the font color.
And I think that this is consistent with other del + ins design I've seen across other websites e.g. GitHub

```diff
- Noisy Miner
+ Brush Turkey
```

Reply:
yeah it might be too much. But I like your suggestion. 

I think it's useful to visibly distinguish a correct. Let's test it with users and see who complains